app/components/sidebars/main/main_sidebar.js

```diff
@@ -131,8 +125,8 @@ export default class ChannelDrawer extends Component {
     };
 
     closeChannelDrawer = () => {
-        if (this.refs.drawer && this.refs.drawer.isOpened()) {
-            this.refs.drawer.close();
+        if (this.drawerOpened) {
+            this.refs.drawer.closeDrawer();
```

Comment:
This causes an issue for me on team switch. 
<img width="395" alt="screen shot 2018-06-25 at 7 38 04 pm" src="https://user-images.githubusercontent.com/4973621/41855094-6a777bea-78af-11e8-9a6d-36b1cd968b86.png">

Reply:
yeah this part of if should be added back

Reply:
will fix